### PR TITLE
Looknfeel

### PR DESCRIPTION
**Pull request by felixDev22**
**In this repo**

- [ ] Create a new branch
- [ ]  Added styling in CSS to create a look and feel of the "race car game"
- [ ] Maintained the wireframe and functionalities

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -12,68 +13,101 @@
...
+  --text-grey: #4b4b4b;
 }
 
 body {
   font-family: Open Sans, sans-serif;
-  background: #dfdfdf;
+}
+
+.game-wrapper {
+  background: url(./images/Car-Race2.png);
```
Comment (pinkmoon25) [inline src/style.css]: - [ ] Good job with the final touches, I think you accidentally used incorrect name for your image file and currently no styles are applied and we are getting this error :point_down: 
image

Please fix your image file name and make sure the webpack compiles the stylesheet successfully :+1:

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @felixDev22 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
Good commits :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Removing the Microverse logo from the Readme, you can add your own logo if you have one otherwise just use the project;s name, It will be more professional and personalised to your projects.
- Please consider adding the steps such as Install and usage instruction in your readme so anybody who visits this project can easily install and run it locally
image

- Please remove this tech stack and replace with the ones used in the project please see :point_down: 
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @felixDev22, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

- **As the last code reviewer said:** 👇🏽
- [x] Please 🙏🏽 remove the Microverse logo from the Readme file and add your own logo if you have one otherwise just use the project;s name, It will be more professional and personalized to your projects.

> Removing the Microverse logo from the Readme, you can add your own logo if you have one otherwise just use the project's name, It will be more professional and personalized to your projects.

- [x] Kindly add the instructions that allow a common user to use your application.

> Please consider adding the steps such as Install and usage instructions in your readme so anybody who visits this project can easily install and run it locally

image

- [x] Please 🙏🏽 replace the tech stack with the ones used in the project.

> Please remove this tech stack and replace it with the ones used in the project please see 👇

image 👇

image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by felixDev22** [comment]
Thank you - I have implemented the readme

**Review by christianbiring1 — APPROVED** [review]
Hi @felixDev22 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
imageswelljob

## Highlights
- [ ] No linter errors :heavy_check_mark: 
- [ ] Correct Git flow :heavy_check_mark:
- [ ] Descriptive PR :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**